jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/udp/AbstractUdpReceiverPublisher.java

```diff
@@ -39,20 +39,17 @@ public void setStopped(boolean stopped) {
       this.stopped = stopped;
    }
 
-   @Autowired
-   public AbstractUdpReceiverPublisher(OdeProperties odeProps, int port, int bufferSize) {
+   protected AbstractUdpReceiverPublisher(OdeProperties odeProps, int port, int bufferSize) {
```

Comment:
note: This is neither a concrete class nor a valid Bean, so the `@Autowired` annotation was unnecessary. the change from `public` to `protected` is intentional since this constructor is only used, and should only be used, by the implementing classes via `super(...)`